### ISSUE TITLE
Doesn't work with Werbstorm 2019.3

### ISSUE THREAD

**Issue by oscarmoreta** [plamen-paskov/serverless-framework-gui#20]
Hello

I can't install this serverless-framework-gui with webstorm 2019.3

Any idea?

thanks in advance

**Comment by SamuelHaleMN** [comment]
Can confirm it doesn't work with the latest 2019.3.1 either.

**Comment by oscarmoreta** [comment]
Yes, I try to install plugin from disk and the process fail with message 'Plugin Serverless Framework GUI is incompatible with this installation.

My system:
Macbook Pro 2017
MacOS Catalina 10.15.1
Webstorm 2019.3.1

Thanks in advance

**Comment by SamuelHaleMN** [comment]
So is this plugin just dead now, or what?

**Comment by SamuelHaleMN** [comment]
Yup. RIP might-have-been-a-potentially-useful-plugin.

You might have been missed? 🤷‍♂
